lib/potassium/recipes/yarn.rb

```diff
@@ -0,0 +1,14 @@
+class Recipes::Yarn < Rails::AppBuilder
+  def create
+    template '../assets/package.json', 'package.json' unless get(:front_end)
+    template '../assets/bin/update.erb', 'bin/update', force: true
+    application "config.assets.paths << Rails.root.join('node_modules')"
+    append_to_file ".gitignore", "node_modules/\n"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.